# this service wraps the transformPointCloud method in TransformListener (both versions).
# if fixed_frame is empty the non-interpolating version is called and time is ignored.

string target_frame
time   target_time 
sensor_msgs/PointCloud pcin
string fixed_frame
---
sensor_msgs/PointCloud pcout